client/src/routes/+layout.svelte

```diff
@@ -155,6 +155,23 @@
       <li>Scenario 1 will result in the cats winning. The app is now fish-free!</li>
       <li>Scenario 2 will result in the catfish winning and cats dropping lower in the food chain…</li>
     </ul>
+
+    <h3 class="mdc-typography--headline5">Credits</h3>
+    <a href="https://github.com/nstringham" rel="noreferrer" target="_blank">Nate Stringham - Team Lead</a> <br />
+    <a href="https://github.com/njumaah" rel="noreferrer" target="_blank">Nathan Jumaah - Deputy Team Lead</a> <br />
+    <a href="https://github.com/Muaad2001" rel="noreferrer" target="_blank">Muaad Abdulaziz - Team Member</a> <br />
+    <a href="https://github.com/SBAsai" rel="noreferrer" target="_blank">Shunsuke Asai - Team Member</a> <br />
+    <a href="https://github.com/zaverbac" rel="noreferrer" target="_blank">Zac Averbach - Team Member</a> <br />
+    <a href="https://github.com/gchamb" rel="noreferrer" target="_blank">Giantte Chambers - Team Member</a> <br />
+    <a href="https://github.com/cmcclusky" rel="noreferrer" target="_blank">Cameron McClusky - Team Member</a> <br />
+    <a href="https://github.com/mrjoshua520" rel="noreferrer" target="_blank">Josh Salkeld - Team Member</a> <br />
+    <a href="https://twitter.com/00_yupia" rel="noreferrer" target="_blank">Yokaih - Artist</a> <br />
```

Comment:
if we capitalize this we should do it on the readme

Reply:
see #197